### PR TITLE
issue: 3977246 Migrate redmine checker pipeline to blossom

### PR DESCRIPTION
**Pull request by NirWolfer**
Create JJB file for redmine checker, all CI should have code representation.

## Description
Migration of redmine checker pipeline to Blossom

##### What
Create JJB file for redmine checker with Blossom parameters

##### Why ?
All CI should have code representation.
issue: 3977246

## Change type
What kind of change does this PR introduce?
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] CI related changes
- [ ] Documentation content changes
- [ ] Tests
- [ ] Other

## Check list
- [ ] Code follows the style de facto guidelines of this project
- [ ] Comments have been inserted in hard to understand places
- [ ] Documentation has been updated (if necessary)
- [ ] Test has been added (if possible)

### CODE REVIEW

File: .ci/redmine_matrix_job.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+job: LibVMA-redmine-issue
+
+registry_host: harbor.mellanox.com
+registry_auth: swx-storage
+
+kubernetes:
+  privileged: true
+  cloud: swx-k8s-spray
+  nodeSelector: 'beta.kubernetes.io/os=linux'
+
```
Comment (dpressle) [inline .ci/redmine_matrix_job.yaml:11]: remove empty lline

File: .ci/redmine_matrix_job.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  - {name: 'ub16.04-base', url: 'harbor.mellanox.com/swx-infra/x86_64/ubuntu16.04/base', arch: 'x86_64'}
+
+steps:
+  - name: Redmine
+    containerSelector:
+      - "{name: 'ub16.04-base', variant: 1}"
+    run: |
+      #!/bin/bash -eExl
+      env
+      sleep 300
```
Comment (dpressle) [inline .ci/redmine_matrix_job.yaml:34]: did you check why we need this?

File: .ci/redmine_matrix_job.yaml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  - {mountPath: /hpc/local/inst/hpc-internal-tools, hostPath: /hpc/local/inst/hpc-internal-tools}
+
+runs_on_dockers:
+# ubuntu 16.04 is used due to script dependency on python2 old print format
+  - {name: 'ub16.04-base', url: 'harbor.mellanox.com/swx-infra/x86_64/ubuntu16.04/base', arch: 'x86_64'}
+
+steps:
+  - name: Redmine
+    containerSelector:
+      - "{name: 'ub16.04-base', variant: 1}"
```
Comment (dpressle) [inline .ci/redmine_matrix_job.yaml:30]: no need for varient we dont run matrix here

File: .ci/redmine_matrix_job.yaml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  module: groovy
+  run: |
+    mail from: "${MAIL_FROM}",
+      mimeType: 'text/html',
+      to: "${MAIL_TO}",
+      subject: 'Redmine check for build ${currentBuild.number}',
+      body: """
+        <p><b>Build url:</b> <a href=${currentBuild.absoluteUrl}>link</a></p>"""
+    sh 'echo "Finish"'
+
```
Comment (dpressle) [inline .ci/redmine_matrix_job.yaml:59]: missing new line

File: .ci/redmine_matrix_job.yaml [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+steps:
+  - name: Redmine
+    containerSelector:
+      - "{name: 'ub16.04-base', variant: 1}"
+    run: |
+      #!/bin/bash -eExl
+      env
+      sleep 300
+
+      export ghprbPullLink=$ghprbPullLink
```
Comment (dpressle) [inline .ci/redmine_matrix_job.yaml:36]: why we need to export these vars? they should be available in here

File: .ci/redmine_matrix_job.yaml [hunk 6]
```diff
@@ -0,0 +1,59 @@
+---
+job: LibVMA-redmine-issue
+
+registry_host: harbor.mellanox.com
+registry_auth: swx-storage
+
+kubernetes:
+  privileged: true
+  cloud: swx-k8s-spray
```
Comment (dpressle) [inline .ci/redmine_matrix_job.yaml:9]: lets use blossom k8s 
  cloud: il-ipp-blossom-prod
  namespace: swx-media

**Review by dpressle — CHANGES_REQUESTED** [review]

File: .ci/redmine_matrix_job.yaml (hunk 2 above)
Comment (NirWolfer) [reply .ci/redmine_matrix_job.yaml:34]: this does not seem to matter at all in the script context, will remove this

File: .ci/redmine_matrix_job.yaml (hunk 6 above)
Comment (NirWolfer) [reply .ci/redmine_matrix_job.yaml]: Swapped to Blossom k8s

**Review by dpressle — APPROVED** [review]

**Pull request merged**